### ISSUE TITLE
Player Web Credentials

### ISSUE THREAD

**Issue by PedroRendeiro** [colloqi/piSignage#118]
When the player is updated the Web credentials are reseted to the default "pi":"pi". This has happened for the last few releases.

**Comment by colloqi** [comment]
Are you still facing this issue? this should not happen

**Comment by PedroRendeiro** [comment]
Yes, just updated to version 2.8.6 and password was set to the default one

**Comment by glepping** [comment]
same happened with 2.8.6 --> 2.9 update

**Comment by hitesh83** [comment]
tested update from 2.7.1 OS full image to 2.9.1 via open source.
1- first boot user name password is set to default one.
2- after first deploy and reboot, password is back to user define.
